include/aie/Dialect/AIEVec/IR/AIEVecOps.td

```diff
@@ -743,4 +743,20 @@ def AIEVec_NegOp:
   let assemblyFormat = "$source attr-dict `:` type($result)";
   let hasVerifier = 0;
 }
+
+def AIEVec_BxorOp:
+  AIEVec_Op<"bxor", [
+    Pure,
+    AllTypesMatch<["lhs", "rhs", "result"]>
+  ]>,
+  Arguments<(ins AnyVector:$lhs, AnyVector:$rhs)>,
+  Results<(outs AnyVector:$result)> {
+  let summary = "AIE vector bitwise xor";
+  let description = [{
+    AMD-specific intrinsic that computes bitwise xor of two vectors and returns the result.
```

Comment:
vector `bxor()` looks like AIE-ML only?

Reply:
Not AIE-ML only. It is also in AIE2p and AIE4. Refer to http://cervino-doc/aie2p/AIE2P_MASTER/intrinsics/index.html and http://cervino-doc/aie4/AIE4_MASTER/intrinsics/index.html